watchman/checks.py

```diff
@@ -22,7 +23,13 @@ def _check_cache(cache_name):
     key = 'django-watchman-{}'.format(uuid.uuid4())
     value = 'django-watchman-{}'.format(uuid.uuid4())
 
-    cache = get_cache(cache_name)
+    # As of Django 1.7, django.core.cache.caches replaces
+    # django.core.cache.get_cache
+    if django.VERSION < (1, 7):
+        cache = django_cache.get_cache(cache_name)
+    else:
+        cache = django_cache.caches[cache_name]
```

Comment:
@justinsacbibit Thanks for this! 

Thinking about this a bit more, I think we should actually move this into a small function in `watchman/utils.py` to keep this function a bit cleaner.

Thinking something like:

```python
# checks.py

from watchman.utils import get_cache

cache = get_cache(cache_name)

# watchman/utils.py
def get_cache:
    if django.version < (1, 7):
        return django_cache.get_cache(cache_name)
    else:
        return django_cache.caches[cache_name]
```